components/gc-featured-link/gc-featured-link-en.html

```diff
@@ -52,17 +52,104 @@ <h2 class="wb-inv">Featured</h2>
 <div class="container">
 	<h3>Code</h3>
 	<pre><code>&lt;section class="provisional gc-featured-link"&gt;
-		&lt;div class="container"&gt;
-			&lt;div class="row"&gt;
-				&lt;div class="col-xs-12"&gt;
-					&lt;h2 class="wb-inv"&gt;Featured&lt;/h2&gt;
-					&lt;p&gt;&lt;a class="stretched-link" href="#"&gt;[Promotion title]&lt;/a&gt;&lt;/p&gt;
-					&lt;p&gt;Long description [optional]&lt;/p&gt;
-				&lt;/div&gt;
+	&lt;div class="container"&gt;
+		&lt;div class="row"&gt;
+			&lt;div class="col-xs-12"&gt;
+				&lt;h2 class="wb-inv"&gt;Featured&lt;/h2&gt;
+				&lt;p&gt;&lt;a class="stretched-link" href="#"&gt;[Promotion title]&lt;/a&gt;&lt;/p&gt;
+				&lt;p&gt;Long description [optional]&lt;/p&gt;
 			&lt;/div&gt;
 		&lt;/div&gt;
-	&lt;/section&gt;
-	</code></pre>
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
+
+	<h3>Customized background color</h3>
+	<p>To customize the background color, add the attribute <code>data-bg-color</code>. The value should be the hexadecimal code of the desired color.</p>
+	<p>The plugin will automatically apply the correct color to the text: <code>#FFFFFF</code> for dark colors, <code>#333333</code> for light colors, <code>#000000</code> for colors in-between.</p>
+	<p class="mrgn-tp-lg">Dark background example:</p>
```

Comment:
We would need to add a notice about the irregular heading structure along with a skip link to the code sample.